src/packages/hslString.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import ColorPicker from "../components/ColorPicker";
+import withColorModel from "../hocs/withColorModel";
+import { ColorModel, ColorPickerBaseProps } from "../types";
+import { equalColorString } from "../utils/compare";
+import { hslStringToHsv, hsvToHslString } from "../utils/convert";
+
+interface Props extends ColorPickerBaseProps<string> {
+  color: string;
+}
+
+const colorModel: ColorModel<string> = {
+  defaultColor: "hsl(0, 0%, 0%)",
+  toHsv: hslStringToHsv,
+  fromHsv: hsvToHslString,
+  equal: equalColorString,
+};
+
+const HslStringColorPicker: React.FC<Partial<Props>> = withColorModel(ColorPicker, colorModel);
```

Comment:
An idea:
What if we move all src/packages/*.tsx files to their folders?

I mean:
```
src/packages/rgb.tsx → rgb/index.tsx
src/packages/hslString.tsx → hslString/index.tsx
...
src/packages/hex.tsx → src/index.tsx
```

Reply:
Update:
Read that first: https://github.com/omgovich/react-colorful/pull/25#discussion_r475801723